class RetryException(Exception):
    pass


class DemoAvailableException(Exception):
    pass
